src/biogeochem/CNFireLi2014Mod.F90

```diff
@@ -658,12 +666,19 @@ subroutine CNFireFluxes (this, bounds, num_soilc, filter_soilc, num_soilp, filte
    integer                        , intent(in)    :: filter_soilc(:) ! filter for soil columns
    integer                        , intent(in)    :: num_soilp       ! number of soil patches in filter
    integer                        , intent(in)    :: filter_soilp(:) ! filter for soil patches
+   integer                        , intent(out)   :: num_actfirep    ! number of active patches on fire in filter
+   integer                        , intent(out)   :: filter_actfirep(:) ! filter for soil patches
+   integer                        , intent(out)   :: num_actfirec    ! number of active columns on fire in filter
+   integer                        , intent(out)   :: filter_actfirec(:) ! filter for soil columns
    type(dgvs_type)                , intent(inout) :: dgvs_inst
    type(cnveg_state_type)         , intent(inout) :: cnveg_state_inst
+   type(soilbiogeochem_carbonflux_type)   , intent(inout) :: soilbiogeochem_carbonflux_inst
    type(cnveg_carbonstate_type)   , intent(inout) :: cnveg_carbonstate_inst
    type(cnveg_carbonflux_type)    , intent(inout) :: cnveg_carbonflux_inst
    type(cnveg_nitrogenstate_type) , intent(in)    :: cnveg_nitrogenstate_inst
    type(cnveg_nitrogenflux_type)  , intent(inout) :: cnveg_nitrogenflux_inst
+!   type(cnveg_cnmatrix_type)   , intent(inout) :: cnveg_carbonstate_inst
```

Comment:
Please remove the commented out line here and any other unnecessary commented out lines in your code.